rules/mech-building/overview.md

```diff
@@ -1,29 +1,21 @@
+## Overview
 
-> Note the distinction between **Parts** and **Components**. The term Part refers to anything used to build a mech, while the term Components refers specifically to Parts that get installed in slots in your mech's frame.
+Mech building is at the heart of the Mechtronica RPG system and as such has some of the most in depth mechanics. To start we'll introduce the key concepts here before you dive into the details.
 
-| Part Type    | Function    |
-| ------------ | ----------- |
-| Frame        | The mech frame is the core superstructure of any mech. |
-| Plating      | Plating protects a mech's components from damage. |
-| Cabin        | Cabins are the center of command and control for a mech. Mechaneers pilot their mechs from inside the cabin.
-| Mainframes   | A mech's mainframe computer provides CPU and interacts with Modules installed in the Cabin. |
-| Modules      | Modules provide special functionality to a mech, typically at the cost of some CPU. There are a large variety of Modules with unique effects. |
-| Power Plants | A mech's power plant generates Power by using up some sort of fuel. |
-| Fuel Tanks   | Fuel tanks store fuel and supply it to power plants. |
-| Coolers      | Coolers provide Cooling to help manage heat generated by other components. |
-| Weapons      | Weapons are for breaking things, like hostile mechs. |
-| Equipment    | Mech attachments that aren't weapons. Includes a variety of tools, storage devices, and other bits and bobs. |
+The mechtronica system is highly modular. It's not like picking a car model and choosing trims and packages—you're building this from the ground up. You will pick a frame which is where all the other parts for your mech will go, but the main thing it determines are how fast your mech is and how many parts it can fit.
 
-{.table}
+Nearly all parts use {% keyword "Resources" glossary %}. Resources (and the related {% keyword "Power Type" glossary %}) are the main constraints on your mech build. You have to make sure you have enough resources for all your {% keyword "Infrastructure Components" glossary %} to operate and enough left over to use your {% keyword "Attachment Components" glossary %} when you need to.
+
+Most infrastructure components can be grouped into "Subsystems". These groups of components interact with each other closely and set constraints on each other. There are three such subsystems:
+- Frame Subsystem: Together, the frame and plating determine speed and provide some cooling
+- Control Subsystem: The control subsystem are constrained by software and related by production/consumption of CPU
+- Power Subsystem: The power subsystem are constrained by fuel type
+
+Another important note is your cabin, the cabin and the frame are the only two components you can only have one of. The cabin is the ultimate weakpoint of your mech, and if it is destroyed, you die (see [Combat]({{ site.url }}/rules/combat)).
```

Comment:
We still have a comma splice in the first sentence here

Reply:
Reworded/fixed